### PR TITLE
remove client counts flag TM-3030

### PR DESCRIPTION
**Pull request by SidelineCory24**
**Feature Flag Part 2 Removal**
[Config File Changes PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2182)

This was deprecated/set to false. Finally removing.

When the `client_counts` flag is set to true, this should not show when under Client Profiles:
image

### CODE REVIEW

File: src/Components/BidderPortfolio/BidderPortfolioPage/BidderPortfolioPage.jsx [hunk 1]
```diff
@@ -53,19 +49,15 @@ class BidderPortfolioPage extends Component {
...
+      classificationsIsLoading,
       classificationsHasErrored, classifications, defaultHandshake, defaultOrdering } = this.props;
     // Here we just want to check that the 'all_clients' prop exists,
     // because we want the nav section to appear
     // even when we reload the counts.
-    let navDataIsLoading = false;
-    if (useClientCounts) {
-      navDataIsLoading = bidderPortfolioCountsIsLoading && !bidderPortfolioCounts.all_clients;
-    }
+    const navDataIsLoading = false;
```
Comment (elizabeth-jimenez) [inline src/Components/BidderPortfolio/BidderPortfolioPage/BidderPortfolioPage.jsx:60]: We might not need `navDataIsLoading ` anymore?

File: src/Components/BidderPortfolio/BidderPortfolioPage/BidderPortfolioPage.jsx [hunk 2]
```diff
@@ -53,19 +49,15 @@ class BidderPortfolioPage extends Component {
...
-    const useClientCounts = getUseClientCounts();
     const { editType } = this.state;
     const { bidderPortfolio, bidderPortfolioIsLoading, cdosLength,
       bidderPortfolioHasErrored, pageSize, queryParamUpdate, pageNumber,
-      bidderPortfolioCounts, bidderPortfolioCountsIsLoading, classificationsIsLoading,
+      classificationsIsLoading,
       classificationsHasErrored, classifications, defaultHandshake, defaultOrdering } = this.props;
     // Here we just want to check that the 'all_clients' prop exists,
     // because we want the nav section to appear
     // even when we reload the counts.
```
Comment (elizabeth-jimenez) [inline src/Components/BidderPortfolio/BidderPortfolioPage/BidderPortfolioPage.jsx:59]: this comment seems counts related so we may be ok to remove?

Comment (SidelineCory24) [reply src/Components/BidderPortfolio/BidderPortfolioPage/BidderPortfolioPage.jsx]: nice catch, removing

File: src/Containers/BidderPortfolio/BidderPortfolio.jsx [hunk 3]
```diff
@@ -35,9 +32,6 @@ class BidderPortfolio extends Component {
   UNSAFE_componentWillMount() {
     if (get(this.props, 'cdos', []).length) {
       this.getBidderPortfolio();
-      if (getUseClientCounts()) {
-        this.props.fetchBidderPortfolioCounts();
```
Comment (elizabeth-jimenez) [inline src/Containers/BidderPortfolio/BidderPortfolio.jsx:39]: Gonna add this as a tech debt ticket: removing fetchBidderPortfolioCounts

Comment (elizabeth-jimenez) [reply src/Containers/BidderPortfolio/BidderPortfolio.jsx:39]: https://metaphase.atlassian.net/browse/TM-3076

**Comment by elizabeth-jimenez** [comment]
Good to go except for verifying navDataIsLoading

File: src/Components/BidderPortfolio/BidderPortfolioPage/BidderPortfolioPage.jsx (hunk 1 above)
Comment (SidelineCory24) [reply src/Components/BidderPortfolio/BidderPortfolioPage/BidderPortfolioPage.jsx:57]: Saw it being used with `isLoading`, but I see I removed the conditional setting for it, removing

**Comment by SidelineCory24** [comment]
> Good to go except for verifying navDataIsLoading

`navDataIsLoading` has been removed

**Review by elizabeth-jimenez — APPROVED** [review]

**Pull request merged**